### PR TITLE
35 ys styling for user profile show page

### PR DESCRIPTION
**Pull request by YoungSong99**
closing #35 
@borvux @jb520 @FadiBaker92 @APatel-AI 
Please be aware that I made updates to application.css and application.html, so there may be potential conflicts when merging.

@MayaS1111
The profile partial has been completed. However, to finalize the yellow-marked section in the attached screenshot, I will need assistance retrieving the role and capacity information from the user model. 

Could you assist with fetching the role and capacity data for users?

Screenshot 2024-10-05 at 2 31 54 AM

### CODE REVIEW

**Comment by MayaS1111** [comment]
@YoungSong99 
THIS LOOKS AMAZING!!!
Yes, I can take care of the role and capacity data.

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ def show
...
 
     case @role
-    when 'Mentor'
+    when 'mentor'
       @mentors_data = CohortMember.where(role: 'Mentor')
-    when 'Mentee'
+    when 'mentee'
       @mentees_data = CohortMember.where(role: 'Mentee')
-    when 'Admin'
+    when 'admin'
```
Comment (MayaS1111) [inline app/controllers/dashboard_controller.rb:11]: Thanks for changing these, I realized it didn't need to be capitalized later in the day😂

File: app/views/profile/_profile.html.erb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+    <div>
+      <b class="text-lg text-[#202020] font-bold"><%= user.first_name %> <%= user.last_name %></b>
+      <p class="text-sm	text-[#202020] mb-2.5"><%= user.title %></p>
+      <div>
+        <div class="flex gap-1 mb-2">
+          <img src="<%= asset_path('icon/office.svg') %>" alt="office" class="max-w-15 max-h-15">
+          <p class="text-xs text-[#202020]">Discovery Partners Institute</p>
+        </div>
+        <!--      <p><%#= user.company %></p> TODO-->
```
Comment (MayaS1111) [inline app/views/profile/_profile.html.erb:22]: Are you able to:
-Put "TODO:" first? 
-Remove spacing in the beginning (happens on a few different lines)

**Review by MayaS1111 — CHANGES_REQUESTED** [review]
Just 2 small changes. But everything else is great!

File: app/views/profile/_profile.html.erb (hunk 2 above)
Comment (YoungSong99) [reply app/views/profile/_profile.html.erb]: Updated :)

**Review by MayaS1111 — APPROVED** [review]

**Review by FadiBaker92 — APPROVED** [review]
Looks great Young!

**Review by borvux — APPROVED** [review]
Looks great to me!

**Pull request merged**